locales-src/en-US/strings.properties

```diff
@@ -113,30 +71,11 @@ pocket_how_it_works=How it works
 pocket_cta_button=Get Pocket
 pocket_cta_text=Save the stories you love in Pocket, and fuel your mind with fascinating reads.
 
-highlights_empty_state=Start browsing, and we’ll show some of the great articles, videos, and other pages you’ve recently visited or bookmarked here.
```

Comment:
https://github.com/mozilla/activity-stream/pull/5104/files#diff-ebefb3093c5ee42688ddb1d8c6b3509bR60 , is there a reason we are keeping edit top sites strings, do we need a bug to track these?